core/templates/search/search_pages_results.html

```diff
@@ -109,15 +109,6 @@ <h4 class="term">{{ paginator.count }} results
 <form class="form-inline search_results_form" action="" method="GET">
   <div class="row searhc_results_option_set">
     <div class="col-md-3">
-      <label for="state">
```

Comment:
is there anywhere deeper in the code, like in the solr searching methods, where state should be removed, too?  Or are you thinking of this as a superficial change, and you want to keep the state searching functionality elsewhere?